src/components/FixedUserList.css

```diff
@@ -0,0 +1,13 @@
+.FixedUserList {
+  --margin: 0.25rem;
+  position: fixed;
```

Comment:
can you use position absolute instead so that its positioned with respect to its parent instead of viewport?